jax/experimental/jax2tf/jax2tf.py

```diff
@@ -854,7 +854,16 @@ def _unexpected_primitive(p: core.Primitive, *args, **kwargs):
 tf_impl[lax.sign_p] = tf.math.sign
 tf_impl[lax.floor_p] = tf.math.floor
 tf_impl[lax.ceil_p] = tf.math.ceil
-tf_impl[lax.round_p] = tf.math.round
+
+def _round(operand):
+  sign = tf.math.sign(operand)
+  operand *= sign
+  floor = tf.math.floor(operand)
+  operand -= floor
+  cond = tf.math.equal(operand, tf.constant(np.array(0.5), operand.dtype))
+  return sign * (tf.where(cond, 2 * operand, tf.math.round(operand)) + floor)
```

Comment:
I find the equality comparison with 0.5 suspicious. Should it be <= ? The 2 * operand also seems strange.

Reply:
`2 * operand` is indeed weird (leftover from a previous attempt where `abs` was not used and the first condition could match several values). I've replaced it by `tf.constant(np.array(1), operand.dtype)` (which is still `2 * operand` in this case, for the relevant values, except that these values are always `0.5`).
The comparison is accurate, it is meant to make rounding consistent when the decimal part is equal to 0.5, when JAX and TF take different arbitrary rounding choices (the former rounds away from 0, and the latter to the nearest even number).